extension/icu/third_party/icu/common/uloc.cpp

```diff
@@ -2139,11 +2139,11 @@ _uloc_strtod(const char *start, char **end) {
     char buf[30];
     double rv;
     if (!gDecimal) {
-        char rep[5];
+        char rep[5] = {};
         /* For machines that decide to change the decimal on you,
         and try to be too smart with localization.
         This normally should be just a '.'. */
-        sprintf(rep, "%+1.1f", 1.0);
+        U_ASSERT(snprintf(rep, 5, "%+1.1f", 1.0) > 0);
```

Comment:
Asserts can be compiled out - I think this might actually make this not run in release mode.

Can you just put an explicit (void) in front of it? e.g. `(void) snprintf(...)`

Reply:
You right! Thanks!